src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -93,45 +63,68 @@
 import ch.njol.skript.util.ExceptionUtils;
 import ch.njol.skript.util.Task;
 import ch.njol.skript.variables.Variables;
-import ch.njol.util.Callback;
 import ch.njol.util.Kleenean;
 import ch.njol.util.NonNullPair;
 import ch.njol.util.StringUtils;
 import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.Callable;
+import java.util.regex.Matcher;
 
 /**
  * @author Peter Güttinger
  */
 final public class ScriptLoader {
-	private ScriptLoader() {}
-	
+	private ScriptLoader() {

```

Comment:
This was perfectly fine.

Reply:
This wasn't on purpose, I made IntelliJ not change lines like this after changing about 50 files already, let me know if you see this expanded anywhere else please